WaterdogPE/src/main/java/cn/stevei5mc/serverhelper/waterdogpe/listener/PlayerListener.java

```diff
@@ -22,4 +23,11 @@ public static void onPlayerChat(PlayerChatEvent event) {
             }
         }
     }
+
+    public static void onDispatchCommand(DispatchCommandEvent event) {
```

Comment:
在 `WaterdogPE/src/main/java/cn/stevei5mc/serverhelper/waterdogpe/listener/PlayerListener.java` 中添加了命令监听功能。建议对记录的命令进行过滤，避免记录敏感信息（如密码等）。

```suggestion
public static void onDispatchCommand(DispatchCommandEvent event) {
    String cmd = event.getCommand().trim();
    // 过滤敏感命令，避免记录密码等敏感信息
    if (cmd.startsWith("login") || cmd.startsWith("register")) {
        main.getLogger().info(event.getSender().getName() + ": §c/[敏感命令已过滤]");
        return;
    }
    
    if (event.getSender().isPlayer()) {
        main.getLogger().info(event.getSender().getName() + ": §c/" + cmd);
    }
}